### PR TITLE
DB-11110 introduce SHOW CREATE VIEW

### PR DESCRIPTION
**Pull request by hatyo**
##  Description
In this PR a new `show create view` sqlshell command is introduced that shows, for a given view, its DDL statement, similar to what `show create table` does for a given table name.

## How to test
Here are some examples:

```sql
create table t1(col1 int);
create view v1 as select * from t1;
show create view v1;
DDL
---
create view v1 as select * from t1;


create view v2 as select * from v1;
show create view v2;
DDL
---
create view v2 as select * from v1;
```

### CODE REVIEW

File: splice_machine/src/main/java/com/splicemachine/derby/procedures/ShowCreateViewProcedure.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    @SuppressFBWarnings(value="SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE", justification="Intentional")
+    public static void SHOW_CREATE_VIEW(String schemaName, String tableName, ResultSet[] resultSet)
+            throws SQLException, StandardException
+    {
+        List<String> ddls = SHOW_CREATE_VIEW_CORE(schemaName, tableName);
+        resultSet[0] = ProcedureUtils.generateResult("DDL", ddls.get(0) + ";");
+    }
+
+    public static List<String> SHOW_CREATE_VIEW_CORE(String schemaName, String viewName) throws SQLException {
```
Comment (jyuanca) [inline splice_machine/src/main/java/com/splicemachine/derby/procedures/ShowCreateViewProcedure.java:57]: If this procedure always returns one string, change the return type from List<String> to String

**Review by jyuanca — CHANGES_REQUESTED** [review]

**Review by ipraznik-splice — APPROVED** [review]

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST SUCCEEDED +1

**Review by jyuanca — APPROVED** [review]

**Comment by bklo94** [comment]
jenkins please test branch @mem

**Comment by cloudspliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/3220/

**Review by carolp-503 — APPROVED** [review]
+1

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/3230/

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST SUCCEEDED +1

**Pull request merged**